engine/src/main/java/org/hibernate/validator/internal/util/ExecutableHelper.java

```diff
@@ -128,6 +119,40 @@ public static String getSignature(String name, Class<?>[] parameterTypes) {
 	}
 
 	/**
+	 * Returns a string representation of an executable with the given name and parameter types in the form
+	 * {@code <name>(<parameterType 0> ...  <parameterType n>)}, e.g. for logging purposes.
+	 *
+	 * @param name the name of the executable
+	 * @param parameterTypes the types of the executable's parameters
+	 *
+	 * @return A string representation of the given executable.
+	 */
+	public static String getExecutableAsString(String name, Class<?>... parameterTypes) {
+		StringBuilder sb = new StringBuilder( name );
```

Comment:
There are cases were we don't have an actual `Executable`, e.g. when configuring a method using the fluent API. Then this one is used to report a non-existing method. But see `ExecutableFormatter` which is doing what you say for cases where we have an executable (and makes invocations of `Log` a bit simpler).